logos/src/logos/main.py

```
@@ -60,55 +76,950 @@ async def start_grpc():
+        else:
+            raise e
+    finally:
+        # Release scheduler resources
+        try:
+            if result.scheduling_stats and result.scheduling_stats.get("request_id"):
+                try:
+                    _pipeline.scheduler.release(
+                        result.model_id,
+                        result.scheduling_stats.get("request_id")
+                    )
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)